### PR TITLE
Merge pull request #216 from HumanBrainProject/adding-mediatypes

### PR DESCRIPTION
**Pull request by olinux**
Adding mediatypes @jagru20 :smile: :+1:

### CODE REVIEW

File: instances/data/contentTypes/gatan.digitalmicrograph2.jsonld [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
   "@type": "https://openminds.ebrains.eu/core/ContentType",
   "fileExtension": null,
-  "name": "application/vnd.neo.ascii.image",
+  "name": "application/vnd.gatan.digitalmicrograph2",
   "relatedMediaType": null,
   "synonym": [
-    "Ascii Image"
+    "Gatan Digital Micrograph 2"
   ]
-}
```
Comment (jagru20) [inline instances/data/contentTypes/gatan.digitalmicrograph2.jsonld:13]: I am not sure, why this is tracked as being renamed this way. The replacement for this is the file neo.ascii.image.jsonld . But this should not be an issue.

File: instances/data/contentTypes/sivic.jsonld [hunk 2]
```diff
@@ -2,12 +2,11 @@
...
+  "@id": "https://openminds.ebrains.eu/instances/contentTypes/application/vnd.sivic",
   "@type": "https://openminds.ebrains.eu/core/ContentType",
   "fileExtension": null,
-  "name": "application/vnd.thevirtualbrain",
+  "name": "application/vnd.sivic",
   "relatedMediaType": null,
-  "synonym": [
-    "TVB"
-  ]
-}
```
Comment (jagru20) [inline instances/data/contentTypes/sivic.jsonld:13]: the replacement for this file is thevirtualbrain.jsonld

**Review by lzehl — APPROVED** [review]
I think sometimes Github makes some weird mappings with the files. I had that a couple of times in the reviews now...

**Comment by lzehl** [comment]
@jagru20 feel free to merge if you are happy

**Review by jagru20 — APPROVED** [review]

**Pull request merged**